src/app/views/feelings/room/room.component.ts

```diff
@@ -1,5 +1,6 @@
 import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
+import { element } from 'protractor';
```

Comment:
remove unused references

Reply:
I never added this, it must have come from compodoc

Reply:
Probably. Somethimes vs also imports stuff automatically